webpack.demo.babel.js

```diff
@@ -12,7 +12,7 @@ export const minifyPlugins = [
     sourceMap: true,
     comments: false,
     mangle: {
-      except: [ 'ReactPlayer' ]
+      except: ['ReactPlayer', 'maestro-videojs-vast']
```

Comment:
why not mangle?

Reply:
when i mangled it broke

Reply:
also its already uglified by the webpack in that plugin